src/components/form/dt-multi-select-button-group/dt-multi-select-button-group.js

```diff
@@ -144,13 +146,42 @@ export class DtMultiSelectButtonGroup extends DtFormBase {
     `;
   }
 
+  _validateRequired() {
+    const { value } = this;
+
+    if (this.required && value) {
```

Comment:
If value is null/undefined and required is true, this won't trigger a required message